Add dummy tunes functions. Change paintScreen() and display() related functions.

MLXXXp opened the pull request:
The ArduboyTunes class was removed to allow external sound libraries to be used.
Its functions have been reimplemented as deprecated dummy functions so that
existing sketches will compile without errors (but will produce no sound).
Working versions of the functions are available in the ArduboyPlaytune library.

MLXXXp committed:
Reimplement tunes functions as dummy functions

The ArduboyTunes class was removed to allow external sound libraries to be used.
Its functions have been reimplemented as deprecated dummy functions so that
existing sketches will compile without errors (but will produce no sound).
Working versions of the functions are available in the ArduboyPlaytune library.

MLXXXp committed:
Change paintScreen and display related functions

paintScreenAndClearImage() has been removed and its functionality has been
included in paintScreen() using conditional tests.

displayAndClear() has been replaced by display(bool clear).

shogerr commented on src/ArduboyCore.cpp:
```diff
@@ -242,54 +242,33 @@ void ArduboyCore::paintScreen(const uint8_t *image)
-// if this function is changed, make sure corresponding changes
-// are made to paintScreenAndClearImage()
-void ArduboyCore::paintScreen(uint8_t image[])
+void ArduboyCore::paintScreen(uint8_t image[], bool clear)
```
We probably want to set a default, (..., bool clear = 0)

MLXXXp replied:
A default of _false_ **is** set in the function prototype in _ArduboyCore.h_. You set default values in the prototype, not the definition.

MLXXXp committed:
Update keywords.txt

Forgot to remove displayAndClear and paintScreenAndClearImage
from keywords.txt
